scripts/mutect2_wdl/unsupported/mutect2_opt.wdl

```diff
@@ -73,6 +73,9 @@ workflow Mutect2 {
   # This is added to every task as padding, should increase if systematically you need more disk for every call
   Int disk_pad = 10 + ceil(size(picard_jar, "GB")) + gatk4_override_size + select_first([emergency_extra_disk,0])
 
+  # These are multipliers to multipler inputs by to make sure we have enough disk to accomadate for possible output sizez
```

Comment:
multipler - > multiple and sizez -> sizes

Reply:
Oops, sorry I should have looked at the final commit. I think you caught sizes but not multipler?